Packages/com.unity.inputsystem/DocCodeSamples.Tests/DocCodeSamples.asmdef

```diff
@@ -4,13 +4,106 @@
     "references": [
         "GUID:75469ad4d38634e559750d17036d5f7c"
     ],
-    "includePlatforms": [],
+    "includePlatforms": [
+        "Editor"
+    ],
     "excludePlatforms": [],
     "allowUnsafeCode": false,
     "overrideReferences": true,
     "precompiledReferences": [],
     "autoReferenced": false,
     "defineConstraints": [],
-    "versionDefines": [],
+    "versionDefines": [
```

Comment:
Why do we need all of these defines on this DocCode asmdef?

Reply:
If not needed I rather see we keep the asmdef to a minimum.

Reply:
Removed, I thought they were necessary, but I was wrong.